package proto_ff;

import "yd_fieldoptions.proto";


message E_AreaArea
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "区域ID"];
	optional int32 m_subType = 2[(yd_fieldoptions.field_cname) = "细分类型"];
	optional int64 m_belongToSceneID = 3[(yd_fieldoptions.field_cname) = "所属场景"];
	optional int32 m_shapeType = 4[(yd_fieldoptions.field_cname) = "形状类型"];
	optional int32 m_isNotice = 5[(yd_fieldoptions.field_cname) = "是否进出提示"];
}


message Sheet_AreaArea
{
	repeated E_AreaArea E_AreaArea_List = 1[(yd_fieldoptions.field_arysize)=32];
}
